.openpublishing.redirection.json

```diff
@@ -1648,10 +1648,6 @@
       "source_path": "docs/desktop-wpf/getting-started/index.md",
       "redirect_url": "https://docs.microsoft.com/visualstudio/get-started/csharp/tutorial-wpf?toc=/dotnet/desktop-wpf/toc.json&bc=/dotnet/breadcrumb/toc.json"
     },
-    {
-      "source_path": "docs/desktop-wpf/index.md",
```

Comment:
I don't get this. There is no index.md file in the desktop-wpf folder. I think you can remove this entry.

Reply:
Oh I see, you're actually removing it. NVM.